### ISSUE TITLE
la:crud failed with ErrorException

### ISSUE THREAD

**Issue by rokde** [dwijitsolutions/laraadmin#8]
I'm doing your documented installation and after migrating the courses migration i wanted to install the crud stuff with `php artisan la:crud courses`. I got this:
```
Creating controller...
Creating model...
Creating views...
Crud::handle exception: exception 'ErrorException' with message 'file_get_contents(/home/rok/workspace/playground/laraadmin/vendor/dwij/laraadmin/src/stubs/views/index.blade.stub): failed to open stream: No such file or directory' in /home/rok/workspace/playground/laraadmin/vendor/dwij/laraadmin/src/CodeGenerator.php:68
Stack trace:
#0 [internal function]: Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'file_get_conten...', '/home/rok/works...', 68, Array)
#1 /home/rok/workspace/playground/laraadmin/vendor/dwij/laraadmin/src/CodeGenerator.php(68): file_get_contents('/home/rok/works...')
#2 /home/rok/workspace/playground/laraadmin/vendor/dwij/laraadmin/src/Commands/Crud.php(53): Dwij\Laraadmin\CodeGenerator::createViews(Object(stdClass), Object(Dwij\Laraadmin\Commands\Crud))
#3 [internal function]: Dwij\Laraadmin\Commands\Crud->handle()
#4 /home/rok/workspace/playground/laraadmin/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#5 /home/rok/workspace/playground/laraadmin/vendor/laravel/framework/src/Illuminate/Console/Command.php(169): Illuminate\Container\Container->call(Array)
#6 /home/rok/workspace/playground/laraadmin/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 /home/rok/workspace/playground/laraadmin/vendor/laravel/framework/src/Illuminate/Console/Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 /home/rok/workspace/playground/laraadmin/vendor/symfony/console/Application.php(791): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 /home/rok/workspace/playground/laraadmin/vendor/symfony/console/Application.php(186): Symfony\Component\Console\Application->doRunCommand(Object(Dwij\Laraadmin\Commands\Crud), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 /home/rok/workspace/playground/laraadmin/vendor/symfony/console/Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 /home/rok/workspace/playground/laraadmin/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 /home/rok/workspace/playground/laraadmin/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#13 {main}

                             
  [ErrorException]           
  Undefined variable: table  

```

**Comment by gdbhosale** [comment]
Hi @rokde 

I have gone through the error. Its very minute mistake.
I have addressed the issue in version 1.0.8

Please update package by `composer update dwij/laraadmin`
Try to update package periodically to get latest bug fixes.

Best Regards,

**Comment by gdbhosale** [comment]
Hi @rokde,

We have completed all necessary bug fixes in installation in version 1.0.10.
Usually you need to run 'composer dump-autoload' to make it perfect.

Im closing this issue now. Please let me know if issue still persists.

Thanks & Regards,

**Comment by sajadali** [comment]
I have a curd already generated, I have add some new feilds in databse, here what i can do  for update curds
